samples/snippets/src/main/java/com/example/spanner/HubbleTransactionsCodeLab.java

```diff
@@ -305,14 +295,17 @@ public void updatesAndReads(
     }
 
     System.out.println("Beginning workload.");
-    int numProcessors = Runtime.getRuntime().availableProcessors() / 2;
-    ExecutorService writeExecutor = Executors.newFixedThreadPool(numProcessors);
-    ExecutorService readExecutor = Executors.newFixedThreadPool(numProcessors);
+    int numProcessorsTemp = NUM_PROCESSORS / 2;
```

Comment:
We can remove this variable, and replace references to it with NUM_PROCESSORS / 2